refactor(linter): fixes in `react/forward-ref-uses-ref` are suggestions

Sysix opened the pull request:
> Fixes: These are changes that will not change your program's behavior. It is generally safe to run them in a pre-lint hook.
> Suggestions: Changes that may change your program's behavior or make a change you may not want (e.g. removing console.logs). These should be reviewed before being committed to your codebase.

https://oxc.rs/docs/guide/usage/linter/automatic-fixes.html

Sysix commented:
* **#11375** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/11375?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#11376** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/11376?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/11376?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=11376).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

camc314 commented:
### Merge activity

* **May 29, 4:20 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
refactor(linter): fixes in `react/forward-ref-uses-ref` are suggestions (#11376)

> Fixes: These are changes that will not change your program's behavior. It is generally safe to run them in a pre-lint hook.
> Suggestions: Changes that may change your program's behavior or make a change you may not want (e.g. removing console.logs). These should be reviewed before being committed to your codebase.

https://oxc.rs/docs/guide/usage/linter/automatic-fixes.html
